docker/web/Dockerfile

```diff
@@ -32,10 +25,16 @@ RUN chmod +x /entrypoint.sh
 
 COPY ./envs/web.template ./envs/web
 
-COPY . /app
+COPY poetry.lock pyproject.toml /app/
 
 WORKDIR /app/
 
+RUN pip install poetry && \
+  poetry config virtualenvs.create false && \
+  poetry install --with deployment
+
+COPY ./ /app

```

Comment:
probably worth keeping consistent on whether we use a trailing / or not throughout the files (the previous `COPY . /app` does not)

Reply:
You need the trailing slash when there's multiple sources specified (e.g. line 28), so needs to be that one we stick with